documentation/interactions/usage.md

```diff
@@ -85,12 +85,12 @@ fun SomeComposable() {
 }
 ```
 
-You can also make use of the `DraggableAppyxComponent` composable, if you have components that allow gestures to control them:
+You can also make use of the `AppyxComponent` composable, if you have components that allow gestures to control them:
 
 ```kotlin
 @Composable
 fun SomeComposable() {
-    DraggableAppyxComponent(
+    AppyxComponent(
```

Comment:
Doesn't really make sense in context since the preceding paragraph (not shown in the diff) is already about `AppyxComponent`. I guess let's just remove this entire block?

Reply:
Also worth noting for the future that this only documents the **appyx-interactions** usage of it (which is correct, since page in docs is about that) but we're not documenting too much about it in the **appyx-navigation** docs either (just basic usage in https://bumble-tech.github.io/appyx/navigation/quick-start/).

Reply:
done